### PR TITLE
website/docs: expand Identification stage docs

### PR DESCRIPTION
**Pull request by BeryJu**
## Details

REPLACE ME

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)
-   [ ] The translation files have been updated (`make i18n-extract`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

File: website/docs/flow/stages/identification/index.md [hunk 1]
```diff
@@ -33,3 +29,29 @@ Requires authentik 2024.1
...
 
 When enabled, any user identifier will be accepted as valid (as long as they match the correct format, i.e. when [User fields](#user-fields) is set to only allow Emails, then the identifier still needs to be an Email). The stage will succeed and the flow will continue to the next stage. Stages like the [Password stage](../password/index.md) and [Email stage](../email/index.mdx) are aware of this "pretend" user and will behave the same as if the user would exist.
+
+## Source settings
+
+Some sources (like the [OAuth Source](../../../../integrations/sources/oauth/) and [SAML Source](../../../../integrations/sources/saml/)) require user interaction. To make these sources available to users, they can be selected in the Identification stage settings, which will show them below the selected [user field](#user-fields).
+
+By default, sources are only shown with their icon, which can be changed with the _Show sources' labels_ option.
+
+Furthermore, it is also possible to unselect any [user field option](#user-fields), which will result in an identification stage, which will only allow users to use any of the configured sources.
```
[the remark this answers is not in the document]
Comment (tanberry) [reply website/docs/flow/stages/identification/index.md:39]: @BeryJu  please double-check that this is still technically accurate.

File: website/docs/flow/stages/identification/index.md [hunk 2]
```diff
@@ -33,3 +29,29 @@ Requires authentik 2024.1
...
+
+:::info
+Starting with authentik 2023.5, when no user fields are selected and only one source is selected, authentik will automatically redirect the user to that source. This only applies when the _Passwordless flow_ option is not configured.
```
Comment (tanberry) [inline website/docs/flow/stages/identification/index.md:42]:
```suggestion
Starting with authentik 2023.5, when no user fields are selected and only one source is selected, authentik will automatically redirect the user to that source. This only applies when the **Passwordless flow** option is *not* configured.
```

Comment (tanberry) [reply website/docs/flow/stages/identification/index.md:42]: Bold text for UI components, options, settings, etc... and italics for actual emphasis, is what our Style Guide now states. Let's dicuss if you prefer non_UI options to be in italics.

File: website/docs/flow/stages/identification/index.md [hunk 3]
```diff
@@ -33,3 +29,29 @@ Requires authentik 2024.1
...
+:::
+
+## Flow settings
+
+## Passwordless flow
+
+See [Passwordless authentication](../authenticator_validate/index.md#passwordless-authentication)
```
Comment (tanberry) [inline website/docs/flow/stages/identification/index.md:49]:
```suggestion
See [Passwordless authentication](../authenticator_validate/index.md#passwordless-authentication).
```

**Review by tanberry — APPROVED** [review]
some formatting, and one reworded sentence that you should check for technical accuracy. :-)

File: website/docs/flow/stages/identification/index.md [hunk 4]
```diff
@@ -33,3 +29,29 @@ Requires authentik 2024.1
 :::
 
 When enabled, any user identifier will be accepted as valid (as long as they match the correct format, i.e. when [User fields](#user-fields) is set to only allow Emails, then the identifier still needs to be an Email). The stage will succeed and the flow will continue to the next stage. Stages like the [Password stage](../password/index.md) and [Email stage](../email/index.mdx) are aware of this "pretend" user and will behave the same as if the user would exist.
+
```
[the remark this answers is not in the document]
Comment (BeryJu) [reply website/docs/flow/stages/identification/index.md:32]: No, that section is still there?

[the remark this answers is not in the document]
Comment (tanberry) [reply website/docs/flow/stages/identification/index.md:32]: Maybe the diff didn't show that section since it wasn't changed...

**Pull request merged**